lib/Graph/Graph.cpp

```diff
@@ -372,7 +372,9 @@ llvm::StringRef Module::uniqueName(llvm::StringRef name,
 
 ConvolutionNode *Function::createConv(llvm::StringRef name, NodeValue input,
                                       size_t depth, size_t kernel,
-                                      size_t stride, size_t pad, size_t group) {
+                                      size_t stride,
+                                      llvm::ArrayRef<size_t> pads,
```

Comment:
do we need to assert on the size of pads?

Reply:
We actually check the size in calculateConvOutputDims function. PaddingTLBR constructor checks the size.